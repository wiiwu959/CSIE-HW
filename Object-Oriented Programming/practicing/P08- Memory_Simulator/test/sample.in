1000
6
Set 0 String H   i
Set 1 int 3158064
Set 4 short 13878
Set 6 char 0
Get 0 String
Get 1000 int