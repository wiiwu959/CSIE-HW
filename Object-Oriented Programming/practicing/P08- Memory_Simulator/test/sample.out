H00066
Violation Access.
